src/osvr/exportheader.cmake.in

```diff
@@ -0,0 +1,112 @@
+/** @file
+    @brief Automatically-generated export header - do not edit!
+
+    @date 2016
+
+    @author
+    Sensics, Inc.
+    <http://sensics.com/osvr>
+*/
+
+// Copyright 2016 Sensics, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef @EXPORT_BASENAME@_EXPORT_H
+#define @EXPORT_BASENAME@_EXPORT_H
+
+#ifdef @EXPORT_BASENAME@_STATIC_DEFINE
+#  define @EXPORT_BASENAME@_EXPORT
+#  define @EXPORT_BASENAME@_NO_EXPORT
+#endif
+
+/* Per-platform defines */
+#if defined(_MSC_VER)
+/* MSVC on Windows */
+
+#ifndef @EXPORT_BASENAME@_EXPORT
+#  ifdef @LIBNAME_FULL@_EXPORTS
+      /* We are building this library */
+#    define @EXPORT_BASENAME@_EXPORT __declspec(dllexport)
+#  else
+      /* We are using this library */
+#    define @EXPORT_BASENAME@_EXPORT __declspec(dllimport)
+#  endif
+#endif
+
+#ifndef @EXPORT_BASENAME@_DEPRECATED
+#  define @EXPORT_BASENAME@_DEPRECATED __declspec(deprecated)
+#endif
+    
+#elif defined(_WIN32) && defined(_GNUC_)
+/* GCC-compatible on Windows */
+
+#ifndef @EXPORT_BASENAME@_EXPORT
+#  ifdef @LIBNAME_FULL@_EXPORTS
+      /* We are building this library */
+#    define @EXPORT_BASENAME@_EXPORT __attribute__((dllexport))
+#  else
+      /* We are using this library */
+#    define @EXPORT_BASENAME@_EXPORT __attribute__((dllimport))
+#  endif
+#endif
+
+#ifndef @EXPORT_BASENAME@_NO_EXPORT
+#  define @EXPORT_BASENAME@_NO_EXPORT __attribute__((visibility("hidden")))
```

Comment:
I don't think you can do `__attribute__((visibility("hidden")))` on Windows, can you? Do you want me to try building this branch on msys/mingw?